File: Templates/Examples/NHibernate/VisualBasic/NHibVbSample.Data/ManagerObjects/CategoryManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports NHibVbSample.Generated.BusinessObjects
Imports NHibVbSample.Generated.Base

Namespace NHibVbSample.Generated.ManagerObjects
	'Public Partial Interface ICategoryManager
	'	Inherits IManagerBase(Of Category, System.String)
	'End Interface

	Partial Class CategoryManager
		Inherits ManagerBase(Of Category, System.String)
		Implements ICategoryManager
	End Class
End Namespace

File: trunk/Projects/Framework-Samples/NHibernate/CSharp/NHibCsSample.Data/ManagerObjects/CategoryManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using NHibCsSample.Generated.BusinessObjects;
using NHibCsSample.Generated.Base;

namespace NHibCsSample.Generated.ManagerObjects
{
    public partial interface ICategoryManager : IManagerBase<Category, string>
    {
	}
	
	partial class CategoryManager : ManagerBase<Category, string>, ICategoryManager
    {
	}
}